src/process-utils.ts

```diff
@@ -0,0 +1,28 @@
+import {exec} from '@actions/exec';
+import {PathLike} from 'fs';
+
+/**
+ * Run actions/exec.exec and return the output from stdout.
+ *
+ * @param cmd command to execute.
+ * @param args arguments for the command.
+ * @param cwd working directory to execute the command in.
+ * @returns promise with the output from stdout.
+ */
+export async function execWithStdoutCap(
```

Comment:
If you have just one function in a file, you can use [`default` exports](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#using_the_default_export). But maybe it won't be a good idea to use it because it gives you the freedom to change names of the class, function, etc. So if two different files use this function, they can use it by two different names. This may cause inconsistency. 

Anyway, I just wanted to run the idea of `default` exports across you.

Reply:
I did not know about default exports, that's a very nice feature. I'm very likely to add more process helpers to this module, and so it's probably not appropriate here, but it's good to know it exists.